data/core/units/monsters/Jinn.cfg

```diff
@@ -11,17 +11,24 @@ units/monsters/jinn#enddef
     image="{IMG_PATH_TEMP}/jinn.png"
     hitpoints=58
     movement_type=spirit
-    movement=8
+    movement=6
     experience=100
     level=2
     alignment=liminal
     advances_to=null
     {AMLA_DEFAULT}
-    cost=56
+    cost=54
     usage=archer
     description=""
     die_sound={SOUND_LIST:HUMAN_OLD_DIE}
     undead_variation=null
+    [resistance]
+        blade=80
+        pierce=70
+        impact=100
+        fire=80
+        cold=100
+    [/resistance]
```

Comment:
I think I'd raise cold a little, to 110 or maybe even 120. A vulnerability to cold seems reasonable for a desert-dwelling creature, doesn't it?

Other than that I don't have any particular objections to these resistances.

Reply:
Reducing cold resistance is fine. I will do that later.